app/actions/ActionTypes.js

```diff
@@ -282,6 +282,7 @@ export const Survey = {
  */
 export const SurveySubmission = {
   FETCH: generateStatuses('Survey.FETCH_SUBMISSIONS'),
+  FETCH_FOR_USER: generateStatuses('Survey.FETCH_FOR_USER'),
```

Comment:
Maybe call this something with Survey or SurveySubmission so that the action log is more obvious?

Reply:
Agree!

I guess `FETCH_SUBMISSIONS` is ok, or is that a bit misleading?

Reply:
I guess these actionTypes should actually be rewritten to `SurveySubmission.FETCH`, etc instead of `Survey.FETCH_SUBMISSION`. I don't actually use this `FETCH_FOR_USER` atm iirc, I just figured the content of the action was separate enough from the `FETCH` that they should be different actions. We could call them `FETCH_ALL` and `FETCH`? I'm just aware that `FETCH` usually does something like `endpoint="someApp/{id}"`, while we're using a filter to do some custom stuff. But I guess that's fine, sine we don't really ever want to do the original kind of single fetch?

Reply:
That sounds reasonable!
`SurveySubmission.FETCH` is indeed better that `Survey.FETCH_XXX`!